templates/member/admin/members_from_ip.html

```diff
@@ -90,4 +90,16 @@
         {% endblocktrans %}
     </p>
     {% endif %}
+
+    {% if ip_is_already_blocked %}
+        <p>
+            <a class="btn btn-cancel" disabled>{% trans "Cette adresse IP est déjà bloquée" %}</a>
+        </p>
```

Comment:
```suggestion
        </p>
        <br>
```

Sinon le tableau en-dessous est collé au bouton.